api/.docker/watch_tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+ptw ereqs_admin omb_eregs reqs
```

Comment:
What do you think about passing through call arguments:
```bash
ptw ereqs_admin omb_eregs reqs "$@"
```
? This would allow us to set flags.

Alternatively, since this is just a one liner, it could be the entrypoint for the docker command. That'd also allow the args to be passed in.

Reply:
Good idea, I'll add that.